infrastructures/openshift/src/main/java/org/eclipse/che/workspace/infrastructure/openshift/project/OpenShiftProject.java

```diff
@@ -68,6 +68,7 @@ public OpenShiftPersistentVolumeClaims persistentVolumeClaims() {
   /** Removes all object except persistent volume claim inside project. */
   public void cleanUp() throws InfrastructureException {
     pods.delete();
+    pods.stopWatch();
```

Comment:
Looks like tests are not modified in accordance with this change

Reply:
same to [this](https://github.com/eclipse/che/pull/6057#discussion_r138059455)